x-pack/plugins/security_solution/public/entity_analytics/components/severity/common/index.tsx

```diff
@@ -7,31 +7,33 @@
 
 import React from 'react';
 
-import { EuiHealth, EuiTextColor, transparentize } from '@elastic/eui';
+import { EuiHealth, EuiTextColor, useEuiTheme } from '@elastic/eui';
 
 import styled, { css } from 'styled-components';
-import { euiLightVars } from '@kbn/ui-theme';
 
-import { RISK_SEVERITY_COLOUR } from '../../../common/utils';
+import { useRiskSeverityColors } from '../../../common/utils';
 import { HoverPopover } from '../../../../common/components/hover_popover';
 import type { RiskSeverity } from '../../../../../common/search_strategy';
 
-const RiskBadge = styled.div<{ $severity: RiskSeverity; $hideBackgroundColor: boolean }>`
+const RiskBadge = styled.div<{
+  $severity: RiskSeverity;
+  $hideBackgroundColor: boolean;
+}>`
   ${({ theme, $severity, $hideBackgroundColor }) => css`
     width: fit-content;
-    padding-right: ${theme.eui.euiSizeS};
+    padding-right: ${theme.eui.euiSizeXS};
```

Comment:
Question: Is this change expected, from `s` (`8px`) to `xs` (`4px`)?

Reply:
Good catch, nope. Will revert.